auto_dctest.go

```diff
@@ -117,7 +111,7 @@ func AutoDCTestEntryPoint(ctx context.Context, m *pubsub.Message) error {
 			builder.Build(),
 		)
 		if err != nil {
-			log.Error("[auto-dctest] failed to create instance(s)", map[string]interface{}{
+			log.Error("failed to create instance(s)", map[string]interface{}{
```

Comment:
When an error occurs, can we notice it?

Reply:
Yes.

This is because instances are always deleted automatically if the startup-script fails and
Compute Engine emits a log which invokes the slack notifier Cloud Function after that.
https://github.com/cybozu-go/neco-gcp/blob/new-compute-log-api/neco.go#L77-L81

Reply:
At this line, an error occurs when the GCP API call is failed.  I think the startup-script isn't executed yet at the time.

Reply:
Oh, you are correct.
But, the logs from any Cloud Function is currently not captured by the slack notifier to avoid an infinite loop.
This means the logs from Cloud Function with "[auto-dctest]" are actually not captured even in the master branch (This is the reason why I deleted "[auto-dctest]"s).

I suppose that Compute Engine emits a log that says "Failed to insert an instance" with "operation.last=true" when this error happens, and that It's better to handle this error to send messages via slack.

Reply:
We should capture a log from Compute Engine API to send a notification when this error happens, IMO.

Reply:
Or allow users to capture logs from Cloud Function except for slack notifier.

I think this approach is simpler and easier, but I'm reluctant to do that because this is more likely to cause an infinite loop...

Reply:
On the discussion with @tapih 
- Logs In the Cloud functions aren't caught by Cloud Logging due to avoid an infinite loop.
- Users are able to notice that something is wrong if the instance is unavailable. So, we decide to remain the codes as is

Reply:
We prefer keeping the possibility for an infinite loop low to having the slack notifier send messages even for less probable errors.